pennylane/qnode.py

```diff
@@ -195,16 +195,23 @@ class QNode:
         func (callable): a Python function containing :class:`~.operation.Operation`
             constructor calls, returning a tuple of :class:`~.operation.Expectation` instances.
         device (:class:`~pennylane._device.Device`): device to execute the function on
+        cache (bool): If ``True``, the quantum function used to generate the QNode will
+            only be called to construct the quantum circuit once, on first execution,
+            and this circuit will be cached for all further executions. Only activate this
```

Comment:
update to match any changes from decorator docstring

Reply:
done